selfdrive/monitoring/helpers.py

```diff
@@ -9,6 +9,7 @@
 from openpilot.common.params import Params
 from openpilot.common.stat_live import RunningStatFilter
 from openpilot.common.transformations.camera import DEVICE_CAMERAS
+from openpilot.system.hardware import HARDWARE
```

Comment:
this should use the sensor type published in `driverCameraState`

Reply:
this better be set at init no? It would be annoying to override DM.settings in the loop..
Also I see we do the same for calibrationd